src/main/java/com/fdmgroup/forex/models/composites/TradeId.java

```diff
@@ -0,0 +1,32 @@
+package com.fdmgroup.forex.models.composites;
```

Comment:
Do we need to add getters and setters? ChatGPT seems to suggest we should?

Reply:
As discussed, we can rely on JPA's handling of the composite primary key via the `@IdClass` annotation, and revisit if needed.